src/couch/src/couch_att.erl

```diff
@@ -708,6 +709,9 @@ upgrade(#att{} = Att) ->
 upgrade(Att) ->
     Att.
 
+to_tuple(#att{name=Name, att_len=Len, type=Type, encoding=Encoding}) ->
```

Comment:
not keen on the name. a record already is a tuple, so I'm guessing the only purpose here is to ignore other fields (and any new fields).

Reply:
The only reason this exists is because `couch_multipart_httpd:length_multipart_stream()` [makes assumptions](https://github.com/apache/couchdb/blob/master/src/couch/src/couch_httpd_multipart.erl#L266) about the format of the stub. This is usually called from `chttpd`. I didn’t find a way around this without exposing `couch_att`’s `att#` record.

But yeah, I’m very okay with any other name here. What would be better?

Reply:
```
[16:26:10]  <+jan____>	rnewson: do you have a better name for to_tuple in https://github.com/apache/couchdb/pull/1253/files#r178029434?
[16:27:14]  <+rnewson>	I don't, proceed with to_tuple.
```